components/raftstore/src/store/fsm/store.rs

```diff
@@ -1412,6 +1407,7 @@ impl<EK: KvEngine, ER: RaftEngine> RaftBatchSystem<EK, ER> {
         let mut builder = RaftPollerBuilder {
             cfg,
             store: meta,
+            store_start_time: time::get_time(),
```

Comment:
I don't think it's the correct start time.

Reply:
Reverted, not applicable anymore